components/card/index.css

```diff
@@ -85,88 +193,106 @@ governing permissions and limitations under the License.
   }
 }
 
+.spectrum-Card-quickActions {
+  inset-inline-start: var(--mod-card-actions-spacing, var(--spectrum-card-actions-spacing));
+  inset-block-start: var(--mod-card-actions-spacing, var(--spectrum-card-actions-spacing));
+}
+
+.spectrum-Card-actions {
+  inset-block-start: var(--mod-card-actions-spacing, var(--spectrum-card-actions-spacing));
+  inset-inline-end: var(--mod-card-actions-spacing, var(--spectrum-card-actions-spacing));
+}
+
 .spectrum-Card-coverPhoto {
-  block-size: var(--spectrum-card-coverphoto-height);
+  block-size: var(--mod-card-preview-minimum-height, var(--spectrum-card-preview-minimum-height));
   box-sizing: border-box;
 
   display: flex;
   align-items: center;
   justify-content: center;
 
-  border-block-end: var(--spectrum-card-coverphoto-border-bottom-size) solid transparent;
-  border-top-left-radius: calc(var(--spectrum-card-border-radius) - 1px);
-  border-top-right-radius: calc(var(--spectrum-card-border-radius) - 1px);
+  border-top-left-radius: var(--mod-card-corner-radius, var(--spectrum-card-corner-radius));
+  border-top-right-radius: var(--mod-card-corner-radius, var(--spectrum-card-corner-radius));
   border-bottom-right-radius: 0;
   border-bottom-left-radius: 0;
 
   background-size: cover;
   background-position: center center;
+  background-color: var(--mod-card-background-color, var(--spectrum-card-background-color));
+  border-block-end-color: var(--mod-card-border-color, var(--spectrum-card-border-color));
 }
 
 .spectrum-Card-body {
-  padding-block-start: var(--spectrum-card-body-padding-top);
-  padding-inline-end: var(--spectrum-card-body-padding-right);
-  padding-block-end: var(--spectrum-card-body-padding-bottom);
-  padding-inline-start: var(--spectrum-card-body-padding-left);
-
-  &:last-child {
-    border-top-left-radius: 0;
-    border-top-right-radius: 0;
-    border-bottom-right-radius: var(--spectrum-card-border-radius);
-    border-bottom-left-radius: var(--spectrum-card-border-radius);
-  }
+  padding-block-start: var(--mod-card-title-padding-top, var(--spectrum-card-title-padding-top));
+  padding-inline-end: var(--mod-card-body-spacing, var(--spectrum-card-body-spacing));
+  padding-inline-start: var(--mod-card-body-spacing, var(--spectrum-card-body-spacing));
+  padding-block-end: var(--mod-card-body-padding-bottom, var(--spectrum-card-body-padding-bottom));
 }
 
 .spectrum-Card-preview {
   overflow: hidden;
-  border-top-left-radius: calc(var(--spectrum-card-border-radius) - 1px);
-  border-top-right-radius: calc(var(--spectrum-card-border-radius) - 1px);
+  border-top-left-radius: var(--mod-card-corner-radius, var(--spectrum-card-corner-radius));
+  border-top-right-radius: var(--mod-card-corner-radius, var(--spectrum-card-corner-radius));
```

Comment:
same thing comment here too, should these be [logical properties for border radius](https://css-tricks.com/css-logical-properties-and-values/#aa-border-radius)